src/main/java/org/cloudfoundry/promregator/cfaccessor/CFAccessor.java

```diff
@@ -21,7 +22,9 @@
 
 	Mono<ListApplicationsResponse> retrieveAllApplicationIdsInSpace(String orgId, String spaceId);
 
-	Mono<GetSpaceSummaryResponse> retrieveSpaceSummary(String spaceId);
+  Mono<GetSpaceSummaryResponse> retrieveSpaceSummary(String spaceId);
```

Comment:
Looks like the space/tab settings in your editor are misaligned.

Reply:
I have added an editorconfig file to make the definition explicit.